File: 3005-count-elements-with-maximum-frequency/3005-count-elements-with-maximum-frequency.java
class Solution {
    public int maxFrequencyElements(int[] nums) {
        Map<Integer,Integer> mp = new HashMap<>();

        for(int i:nums){
            mp.put(i,mp.getOrDefault(i,0)+1);
        }
        int maxfreq=0;
        for(int i:mp.values()){
          maxfreq=Math.max(maxfreq,i);
        }
        int sum=0;
        for(int i:mp.values()){
            if(maxfreq==i){
                sum=sum+i;
            }
        }
        return sum;
    }
}

File: 3005-count-elements-with-maximum-frequency/3005-count-elements-with-maximum-frequency.cpp
class Solution {
public:
    int maxFrequencyElements(vector<int>& nums) {
        map<int,int>mp;
        for(int i:nums){
            mp[i]++;
        }
        int maxfreq=0;
        for(auto i:mp){
          maxfreq=max(maxfreq,i.second);
        }
        int sum=0;
        for(auto i:mp){
            if(i.second==maxfreq){
                sum=sum+i.second;
            }
        }
        return sum;
    }
};